apps/drupal-default/particle_theme/templates/block/block--utc-hero--full.html.twig

```diff
@@ -37,18 +37,19 @@
 	{% endif %}
 	{# Sets colors for hero content depending on selection #}
 	{% if utc_hero.hero_color == 'Dark Blue' %}
-		{% set title_color = 'lg:text-white' %}
-		{% set body_color = 'lg:text-white' %}
-		{% set tag_color = 'lg:text-white' %}
-		{% set btn_border_color = "lg:border-white" %}
+		{% set title_color = 'text-white' %}
+		{% set body_color = 'text-white' %}
+		{% set tag_color = 'text-white' %}
+		{% set btn_border_color = "border-white" %}
 		{% set bg_color = 'background-color:#112e51' %}
```

Comment:
should this use a utc theme variable instead of the color hex?

Reply:
I'm guessing Brittany was just following the off-theme formatting of the light gray. I can change to theme variable.

Reply:
#112e51 is the dark blue.

Reply:
Ok, the reason the hex color is being used is because the user has a choice between light gray and dark blue. The light gray is not a theme variable, so it has to be implemented through the style attribute. That dictates how the dark blue is applied. It has to be through the style attribute as well. The theme variables are implemented through the class attribute, which won't work for the light gray. However, I can make it a theme variable, but that color is only to be used for the hero background. Thoughts?